api/v1alpha1/openapi.yaml

```diff
@@ -150,6 +150,51 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/Error'
+  /api/v1/sources/{id}/image:
+    get:
+      tags:
+        - source
+        - image
+      description: get the OVA file for the source
+      operationId: getSourceImage
+      parameters:
+        - name: id
+          in: path
+          description: ID of the source
+          required: true
+          schema:
+            type: string
+        - name: ip
+          in: query
+          description: IP of the agent server
+          schema:
+            type: string
```

Comment:
Is this the IP for the service?  This should be taken from the service deployment (which we don't have yet).

Reply:
Right. I've added this for developement purposes, so user can generate OVA for his own env for testing.

Reply:
Ill remove it and will add env variable